patches/BetterReflection.patch

```diff
@@ -0,0 +1,11 @@
+--- src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php
++++ src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php
+@@ -590,7 +590,7 @@ final class PhpStormStubsSourceStubber implements SourceStubber
+     {
+         $newStmts = [];
+         foreach ($stmts as $stmt) {
+-            assert($stmt instanceof Node\Stmt\ClassConst || $stmt instanceof Node\Stmt\Property || $stmt instanceof Node\Stmt\ClassMethod);
```

Comment:
A fix need to be done in `ondrejmirtes/better-reflection` because `@since` is also use on enum
cf https://github.com/JetBrains/phpstorm-stubs/commit/2630c4604859a43fe761108360f4b4352056aa74#diff-ef8aa905e0656e00a12783a18d25215a90772b1e75e3a18f3e4815ef9c68ccf4R41

but since it's a fork of `Roave/BetterReflection` I'm not sure how you deal with such situation @ondrejmirtes :
- Should I open a PR on your repo ?
- Should I open a PR on `Roave/BetterReflection` directly ? And you'll cherry-pick after the merge on `Roave/BetterReflection` ?
- Should I do both ?

Reply:
Just open it on Roave, I regularly rebase on top of them.

Reply:
My fix is merged on Roave https://github.com/Roave/BetterReflection/commit/31b5dd11bdf548a1112a5c7fd287226c4dacaa1e

Now, if you have time @ondrejmirtes, I need a rebase on your fork in order to finish this PR :)

Reply:
@VincentLanglet Done, you can rebase and continue your work :)

Reply:
Thanks, I think the PR is ready then.